File: cpp/ACMP33.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main() {
    int garry, larry, together;
    cin >> garry >> larry;
    together = min(10, garry + larry - 1);
    cout << (together - garry) << ' ' << (together - larry) << endl;

    return 0;
}

File: src/ACMP33.java
import java.io.PrintWriter;
import java.util.Scanner;

public class ACMP33 {
    // your solution here
    private void solve(Scanner in, PrintWriter out) {
        int garry = in.nextInt(),
            larry = in.nextInt();

        int together = Math.min(10, (garry + larry) - 1);
        out.println((together - garry) + " " + (together - larry));
    }

    void run() {
        try (
            Scanner in = new Scanner(System.in);
            PrintWriter out = new PrintWriter(System.out)
        ) {
            solve(in, out);
        }
    }

    public static void main(String[] args) {
        new ACMP33().run();
    }
}
